frontend/packages/dev-console/src/components/topology/topology-utils.ts

```diff
@@ -693,7 +693,7 @@ export const topologyModelFromDataModel = (
       data.groupResources = d.children && d.children.map((id) => dataModel.topology[id]);
       return {
         width: 300,
-        height: 100,
+        height: d.type === TYPE_KNATIVE_SERVICE ? 100 : 180,
```

Comment:
We should look at making this height dynamic based on the number of different types we will have. Probably worth a separate issue.
Or we update topology to better support dynamic height nodes based on the rendering.